src/main/java/com/demo/controller/BaseController.java

```diff
@@ -22,18 +26,26 @@ public String root() {
 	 */
 	@RequestMapping("/index")
 	public String index(Model model){
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Rename method "index" to prevent any misunderstanding/clash with field "INDEX" defined on line 17. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1845)